components/Buttons/src/MDCButton.m

```diff
@@ -215,6 +215,12 @@ - (void)encodeWithCoder:(NSCoder *)aCoder {
   [aCoder encodeObject:_accessibilityLabelForState forKey:MDCButtonAccessibilityLabelsKey];
 }
 
++ (void)initialize {
+  // Default background colors.
+  [[MDCButton appearance] setBackgroundColor:MDCColorFromRGB(MDCButtonDefaultBackgroundColor)
```

Comment:
When cutting the release I see that some internal screenshot tests are showing a change in button background color from white to blue. My read of this code is that this should not have happened but it has. I think we may have to roll these MDCButton changes back?